### ISSUE TITLE
Training of detection network

### ISSUE THREAD

**Issue by fzp8656342** [liuqk3/OTCD#4]
Hi, 
  Thanks for the nice work, I have few questions about the training of the detection network. I try to run your training script of the detection network, which is train_detection_on_mot.py in useful_scripts. However, the loss printed on the console seems abnormal, in the first epoch, the rcnn's regression loss is 100x smaller than three other losses which are rpn_cls, rpn_box and rcnn_cls. How do you train the RFCN network? Does the training process of RFCN similar to Faster-RCNN with multiple steps?
